javagen/src/main/java/com/azure/autorest/util/CodeNamer.java

```diff
@@ -297,6 +297,8 @@ private static String removeInvalidCharacters(String name, char... allowerCharac
         for (Character c : name.toCharArray()) {
             if (Character.isLetterOrDigit(c) || allowed.contains(c)) {
                 builder.append(c);
+            } else {
+                builder.append("_");
```

Comment:
This one seems causing a bit of problem.

Previously genereated enum is such
` SYSTEM_ASSIGNED_USER_ASSIGNED("SystemAssigned, UserAssigned")`

Now it is such
`SYSTEM_ASSIGNED__USER_ASSIGNED("SystemAssigned, UserAssigned")` , one more `_`
Any idea what best to circumvent this?